frontend/src/metabase/components/TextInput/TextInput.tsx

```diff
@@ -54,7 +57,8 @@ function TextInput({
     <TextInputRoot className={className}>
       {icon && <IconWrapper>{icon}</IconWrapper>}
       <Input
-        innerRef={innerRef}
+        as={as}
```

Comment:
Why do we need `as` here?

Reply:
Good catch, I removed this accidental `as`, thanks